packages/runner/package.json

```diff
@@ -8,9 +8,9 @@
     "build-prod": "cross-env NODE_ENV=production yarn build",
     "clean-deps": "rm -rf node_modules",
     "postinstall": "echo '@packages/runner needs: yarn build'",
-    "test": "yarn test-unit",
+    "test": "$(yarn bin)/mocha --config test/.mocharc.json",
```

Comment:
without prefixing `yarn bin` this will use mocha from project root, not v7 installed in this package. Strange